sos/cleaner/mappings/hostname_map.py

```diff
@@ -87,7 +87,7 @@ def get_regex_result(self, item):
         """
         if '.' in item:
             item = item.replace('.', '(\\.|_)')
-        return re.compile(item, re.I)
+        return super(SoSHostnameMap, self).get_regex_result(item)
```

Comment:
If you want to return that way, perhaps add:

`# pylint: disable=super-with-arguments`

To the end of the line?

Reply:
or better, and then we don't need to disable it :)

```python
return super().get_regex_result(item)
```